### PR TITLE
add multisarch and cloudbuild

### LINKED ISSUE #34: Support multi arch / fat manifest
It would be nice to support a multi arch build. 
Could get inspiration from https://github.com/kubernetes/node-problem-detector/pull/336

### PR DESCRIPTION
**Pull request by Sh4d1**
Fixes #34 and #19 

Signed-off-by: Patrik Cyvoct <patrik@ptrk.io>

### CODE REVIEW

**Comment by Sh4d1** [comment]
@cheftako can be merged once https://github.com/kubernetes/test-infra/pull/15137 is merged

**Comment by cheftako** [comment]
/lgtm
Would like to see if @BenTheElder can review as well?

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+timeout: 1200s
+options:
+  substitution_option: ALLOW_LOOSE
+steps:
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20190906-745fed4'
+    entrypoint: make
+    env:
+    - DOCKER_CLI_EXPERIMENTAL=enabled
```
Comment (BenTheElder) [inline cloudbuild.yaml]: you might want to enable this via the makefile instead

**Comment by BenTheElder** [comment]
generally LGTM, I think it may be easier to invoke locally if you enable the experimental CLI env in the makefile instead though.

File: cloudbuild.yaml (hunk 1 above)
Comment (Sh4d1) [reply cloudbuild.yaml]: Yep, I'll do that in a few minutes!

**Comment by Sh4d1** [comment]
@cheftako all good

**Pull request merged**